docs/references/docker-env-vars.md

```diff
@@ -0,0 +1,89 @@
+---
+id: docker-env-vars
```

Comment:
The id and the file name should probably be `web-ui-server-env-vars` - as that focuses on the web ui - and generalizes the env vars (because they could be docker or container or other envs)